examples/complete/variables.tf

```diff
@@ -0,0 +1,68 @@
+variable "namespace" {
+  type        = "string"
+  description = "Namespace (e.g. `cp` or `cloudposse`)"
```

Comment:
```suggestion
  description = "Namespace (e.g. `eg` or `cp`)"
```

Reply:
For the example, we need to provide all default values so the example could be run (e.g. on CI/CD) without any modifications.
We also discussed with @osterman  that for all examples we should remove all defaults from vars and use `tfvars` files.